adding a paper link to Pose Estimation

cuevhv opened the pull request:
Adding First-Person Pose Recognition using Egocentric Workspaces paper from CVPR 2015.

cuevhv committed:
adding a paper link to Pose Estimation

Adding First-Person Pose Recognition using Egocentric Workspaces paper from CVPR 2015.

Sid2697 commented:
Hello @cuevhv, thank you for spending time in improving the list. 

Can you also add it to the list of papers published in CVPR? I will merge it once that is done. Thanks!

cuevhv committed:
added pose estimation paper to CVPR list

Added First-Person Pose Recognition using Egocentric Workspaces to CVPR list. Before it was only in the pose estimation list.

cuevhv commented:
@Sid2697 You are welcome. I have added the paper to the CVPR list as well.
